client/components/widget/query/query-editor-directive.html

```diff
@@ -126,6 +126,20 @@
           </div>
         </div>
       </div>
+
+      <div ng-show="queryEditorCtrl.datasource.view.sort_columns">
+        <div class="sidebar-item">
+          <div class="sidebar-item-label">sort order</div>
+          <div class="sidebar-item-value">
+            <select class="form-control"
+                   ng-model="queryEditorCtrl.datasource.view.sort_column_order">
+              <option value="" selected>Ascending (default)</option>
```

Comment:
"sort order" is blank when I view this page, which makes having two "Ascending" options a bit confusing...